src/services/resource-service.ts

```diff
@@ -122,11 +122,7 @@ export const ResourceService = {
       })
     })
   },
-  getAttachments: async (
-    params?: GetResourcesParams
-  ): Promise<AxiosResponse<ItemsWithCount<Attachment>>> =>
-    await axiosClient.get(URLs.resources.attachments.get, { params }),
-  getAttachmentsQuery: (params?: GetResourcesParams) => {
+  getAttachments: (params?: GetResourcesParams) => {
```

Comment:
Should those `params` be required?
```suggestion
  getAttachments: (params: GetResourcesParams) => {
```

Reply:
Good think you noticed that. Tested on backend. It appeared that fileName param is actually required. But it was optional in frontend code (GetResourcesParams type). So now params are obviously required here